src/main/groovy/com/profitbricks/sdk/model/Request.groovy

```diff
@@ -0,0 +1,36 @@
+package com.profitbricks.sdk.model
+
+import com.profitbricks.sdk.annotation.*
+import groovy.transform.*
+
+/**
+ * a Request POGO
+ * see: https://devops.profitbricks.com/api/cloud/v3/#lans
+ *
+ * Created by ali bazlamit on 30/03/17.
+ * Copyright (c) 2017, ProfitBricks GmbH
+ */
+@ToString(includeNames = true, ignoreNulls = true, includeSuperProperties = true, includePackage = false, excludes = ['resource'])
+@EqualsAndHashCode(callSuper = true)
+final class Request extends ModelBase {
```

Comment:
The idea was to provide the request API endpoints, for example with this you can use the endpoint that lists all requests and you can use the [function ](https://github.com/StackPointCloud/profitbricks-sdk-groovy/blob/898a48b802a0c823c52c764b67f5de2228312ca6/src/main/groovy/com/profitbricks/sdk/Commands.groovy#L234-L238)`requestStatus ` to get the status of a specific request